app/Rules/GoodBye.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Str;
+
+class GoodBye implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        return Str::is($value, 'goodbye') or Str::is($value, 'Goodbye');
+    }
+
+    /**
+     * Get the validation error message.
+     *
+     * @return string
+     */
+    public function message()
+    {
+        return ('You must write goodbye to confirm the deletion of your account');
```

Comment:
The error message should be translatable by our translators. Use `__('validation.goodbye')`, and add this message in `resources/land/en/validation.php`.

Also, `goodbye` should be a [parameter](https://laravel.com/docs/8.x/localization#replacing-parameters-in-translation-strings) so that our translators don't translate it. So in `validation.php`

```php
'goodbye' => 'You must write :code to confirm the deletion of your account',
```

And in `GoodBye.php`
```php
__('validation.goodbye', ['code' => 'goodbye'])
```

Reply:
Changes made!